tests/_data/manifests/src/integrator-manifests-master/TestIntegratorGlossary/1.0.3/installer-manifest.json

```diff
@@ -0,0 +1,14 @@
+{
+    "glossary-key": {
+        "new": {
+            "shopping_list.pre.check.product_merchant_inactive_test": {
```

Comment:
Using `product_merchant_inactive_new` and `product_merchant_inactive_existing` might be better for test names here since it can then be used in the tests more speaking.